index.php

```diff
@@ -39,9 +40,9 @@
 		$notifications_to_delete[] = $response->getUuid();
 	} elseif ( $response->shouldRetry() ) {
 		$notifications_to_retry[] = $response->getUuid();
-	} elseif ( $response->shouldUnsubscribeDevice() ) {
-		$tokens_to_delete[] = $response->getToken();
-	}
+	}// elseif ( $response->shouldUnsubscribeDevice() ) {
```

Comment:
I commented this out for now until we decide what to do with it. I originally had `APNClient` maintaining a mapping of UUIDs to tokens, then applying it to the response object. We _could_ still do that, but the current WP.com implementation maintains that map outside of the client, which IMHO keeps the client cleaner, but then it's the responsibility of the caller to figure out how to manage it. 

I can see good arguments for including that functionality right in the library, but for now, this isn't critical (and could be added down the road).

Reply:
Sounds reasonable 👍 .

At this point, though, I'd question whether it's worth keeping this code -and the related `tokens_to_delete` var which then gets printed at the end of the scrip- or we could just have a comment about that functionality or an issue here on GitHub to keep this idea from getting lost?

I'm fine either way, but not having commented code would make me happier 😄 😅 .

Reply:
Ya – solid point. Fixed in fd2761513b492b4d65146e9f64a197718f6e4cba